src/common.py

```diff
@@ -6,3 +6,9 @@ def get_log(name):
     log = logging.getLogger(os.path.basename(name))
     logging.basicConfig(level=logging.INFO)
     return log
+
+
+def flip_spending_sign(records):
```

Comment:
I'm confused why we need this, I think the parser already flips the sign of amex in `parse_amex`, linked below:
https://github.com/velicanu/expenses/blob/f926c0f25635e9a93261c931aa8caa0f37ba9517/src/parse.py#L17

Reply:
I think expenses as positive is usually the convention for these programs. That's also what intuitively makes sense to me. Since the program is called expenses, i think have expenses be positive values is more natural :P. This function allows me to flip that while allowing you to use your conventions